src/execution_log.rs

```diff
@@ -37,6 +37,7 @@ pub enum ExecutionEvent<H> {
 	SubmitAndWatchResult(SystemTime, Result<(), String>),
 	TxPoolEvent(SystemTime, TransactionStatus<H>),
 	FinalizedMonitor(SystemTime, H),
+	MortalDroppedMonitor(SystemTime, u64),
```

Comment:
Would be nice to dump some numbers in `make_stats`. They are super useful for testing.

Reply:
Will do :+1:

Reply:
[03c2b91](https://github.com/michalkucharczyk/tx-test-tool/pull/43/commits/03c2b9164b5f6c74a7f47bcf944c4b9a6e89cb2d)